### DISCUSSION TITLE
Could you tell me the direct download directory?

### DISCUSSION THREAD

**Discussion by zzzgit** [coqui-ai/TTS#2506] [General Q&A]
<img width="568" alt="image" src="https://user-images.githubusercontent.com/1060733/231501576-c47c3af9-81b8-45f0-ac83-a33e7c6322a9.png">

It's too slow to download in the terminal.

Could you tell me the direct download URL so that I can download one model via other tools?

**Comment by danablend** [comment]
https://github.com/coqui-ai/TTS/blob/dev/TTS/.models.json
